src/legacy/core_plugins/kbn_doc_views/public/views/table.js

```diff
@@ -69,6 +73,16 @@ DocViewsRegistryProvider.register(function () {
           const value = $scope.flattened[field];
           return Array.isArray(value) && typeof value[0] === 'object';
         };
+
+        $scope.enableDocValueCollapse = function (docValueField) {
+          const html = (typeof $scope.formatted[docValueField] === 'undefined') ?
+            $scope.hit[docValueField] : $scope.formatted[docValueField] || $scope.trustAsHtml;
```

Comment:
What's this `$scope.trustAsHtml` here?

Reply:
seems like it's inspired from the template, where it is a filter

```
        <div class="kbnDocViewer__value" ng-bind-html="typeof(formatted[field]) === 'undefined' ? hit[field] : formatted[field] | trustAsHtml"></div>
```
but makes no sense here

Reply:
Can we remove it? Just the name makes me uneasy 😅

Otherwise LGTM

Reply:
I've removed the trusAsHtml